src/element-internals.ts

```diff
@@ -126,7 +126,11 @@ export class ElementInternals implements IElementInternals {
       return undefined;
     }
     const ref = refMap.get(this);
-    refValueMap.set(ref, value);
+    if (value === null){
+      refValueMap.delete(ref);
```

Comment:
Should this be `refValueMap.set(ref, null)‘?

Reply:
I’ll double check if it reproduces the same result.

Reply:
Originally this does refValueMap.set(ref,null) but this ends up being ref=“undefined” in the form data.  So I’m removing the ref if the value is null otherwise just set it to requested value.